### DISCUSSION TITLE
Temperature not saved

### DISCUSSION THREAD

**Discussion by ithax-wb** [open-webui/open-webui#2019] [General]
# Bug Report
As an user the temperature is not saved after changing the default.

## Description

**Bug Summary:**
As an user the temperature is not saved after changing the default. This functionality works properly as admin.

**Steps to Reproduce:**
Login as user
Klick on Name, Setting. In general setting klick "Show" at Advanced Parameters. Change Temperature from default to 0.1. Klick "Save".
Close settings menu.
Klick on Name, Setting. In general setting klick "Show" at Advanced Parameters.  The Temparature is back to default.

**Expected Behavior:**
The temparature should be at the value (e.g. 0.1) and the model should with this value.

**Actual Behavior:**
The Temparature is back to default.

## Environment
- **Open WebUI Version: v0.1.122
- **Ollama (if applicable): 0.1.30
- **Operating System: Ubuntu 20.04
- **Browser (if applicable): Chrome 117.0.5938.157

## Reproduction Details

**Confirmation:**
- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.

**Comment by tjbck** [comment]
Unable to reproduce the issue with users with user role. FYI, advanced params are saved on the browser side. Hope that helps.

**Reply by ithax-wb** [reply]
I have tested it out on a freshly installed browser and indeed it works correct with a user role. My settings in my default chrome browser, not storing cookies, seems to created the problem. Thank you very much for pointing this out to me!

**Comment by spammenotinoz** [comment]
> Unable to reproduce the issue with users with user role. FYI, advanced params are saved on the browser side. Hope that helps.
 Ahh thanks for pointing this out, I have been trying "unsuccessfully to work out why Ollama keeps unloading models, despite keep alive being set to -1, and here is the answer.

**Comment by thusinh1969** [comment]
It would be great if docker can load a config file that apply all default parameters (system prompt, temperature, top_k etc...). New users have to do so much and it becomes teddious while Open WebUI is supposed to be simple.

Thanks,
Steve
